error.go

```diff
@@ -109,6 +112,32 @@ func (e Error) Temporary() bool {
 		e.Timeout()
 }
 
+// Retriable returns true if the error is one the "RETRIABLE" error codes from:
+// https://kafka.apache.org/protocol#protocol_error_codes
+func (e Error) Retriable() bool {
```

Comment:
Is this different than `Error.Temporary`?

Reply:
@abuchanan-nr I'm using that now